pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -4883,3 +4642,17 @@ var _ = Describe("Function getNumberOfPodInterfaces()", func() {
 	})
 
 })
+
+func createVmiAdmissionReview(vmi *v1.VirtualMachineInstance) *admissionv1.AdmissionReview {
+	vmiBytes, err := json.Marshal(&vmi)
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
Please consider using `ExpectWithOffset`
```suggestion
	ExpectWithOffset(1, err).ToNot(HaveOccurred())
```

Reply:
Done